model/vfs/couchdb_indexer.go

```diff
@@ -97,6 +97,9 @@ func (c *couchdbIndexer) prepareFileDoc(doc *FileDoc) error {
 	// taken on camera were the clock was never configured (e.g. 1970-01-01).
 	if date, ok := doc.Metadata["datetime"].(time.Time); ok && date.Year() > 1990 {
 		doc.CreatedAt = date
+		if doc.UpdatedAt.Before(date) {
+			doc.UpdatedAt = date
```

Comment:
Well, it is a difficult question, and I don't have a good answer. `created_at` and `updated_at` are date/times from the client. We also have two date/times from the server (in cozyMetadata). I'd prefer to avoid comparing date/times from the client with a date/time from the server (the now). I don't know what would work better for a smartphone or camera with a clock in the future. At least, by keeping the datetime of the client, we don't lose the order on a couple photos taken on the same device.